ui/app/components/job-editor.js

```diff
@@ -24,6 +24,15 @@ export default class JobEditor extends Component {
     if (this.definition) {
       this.setDefinitionOnModel();
     }
+
+    if (this.args.variables) {
+      this.args.job.set(
+        '_newDefinitionVariables',
+        this.jsonToHcl(this.args.variables.flags).concat(
+          this.args.variables.literal
```

Comment:
`nit`: reverse the order. There is a chance for dupes and we don't care about dupes showing, but we'd rather prioritize the literal over the flags.